tools/packaging/static-build/qemu/Dockerfile

```diff
@@ -59,14 +60,19 @@ ARG BUILD_SUFFIX
 ARG QEMU_DESTDIR
 ARG QEMU_TARBALL
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN git clone  https://github.com/axboe/liburing/ ~/liburing && \
+    cd ~/liburing && \
+    git checkout tags/liburing-2.1 && \
+    make && make install && ldconfig
```

Comment:
What happens when the host kernel doesn't have iouring support?

Reply:
yes, will raise a follow-up PR to update the qemu usage. iouring was added in 5.1, so its should be mostly available unless you are using a kernel older than that. We can check for its availability, and fallback to another I/O mechanism.